pages/treeherder.py

```diff
@@ -115,6 +119,9 @@ class ResultSet(Region):
         _runnable_jobs_locator = (By.CSS_SELECTOR, '.runnable-job-btn.filter-shown')
         _set_bottom_of_range_locator = (By.CSS_SELECTOR, '.open ul > li:nth-child(8) > a')
         _set_top_of_range_locator = (By.CSS_SELECTOR, '.open ul > li:nth-child(7) > a')
+        _test_busted_job_title_locator = (By.CSS_SELECTOR, 'span.group-job-list .btn-purple')
```

Comment:
Maybe it will be better to have CSS_SELECTOR like `button.btn-red`.
What do you think?